masterlist.yaml

```
@@ -6644,27 +6644,64 @@ plugins:
       - crc: 0xA0C60505
         util: 'TES4Edit v4.0.3'
 
+  - name: '(Kragenir''s Death Quest|KDQ - Rural Line Additions|KragenirsDeathQuest-LetThePeopleDrink patch)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/26219/'
+        name: 'Kragenirs Death Quest'
   - name: 'Kragenir''s Death Quest.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/26219/' ]
+    req: [ *SilentVoice ]
+    inc:
+      - '300_White Stallion 4.esp'
```

One last thing: since the mod's compatibility notes mention the old, unsupported patch potentially working for some users, the incompatibility should be disabled when it's used just in case. However, we shouldn't inform users of the patch since it's unsupported.

```suggestion
      - name: '300_White Stallion 4.esp'
        condition: 'not file("Kragenirs_WhiteStallion_Compat.esp")'
```